Data/Sys/GameSettings/GSWD64.ini

```diff
@@ -0,0 +1,29 @@
+# GSWD64 - Star Wars: Rogue Leader
+
+[Core]
+# Values set here will override the main dolphin settings.
+MMU = 1
+FastDiscSpeed = 1
```

Comment:
I can change it but i think pr #1778 should be merged first before doing that or in the meanwhile the game won't work if the opposite happens. 
@JosJuice would a note suffice then? I would like a few more opinions though, i wrote about it at the forum too and only jmc answered, most people didn't seem to care to answer.